### PR TITLE
1. fix "func NewClient" and "func NewInlineClient" without options.

### PR DESCRIPTION
**Pull request by chenji1990**

### CODE REVIEW

**Comment by mochi-co** [comment]
Hi @chenji1990! Thanks for the PR. I was just refactoring this early this morning as there are a couple of other areas this touches on that were overlooked (eg. maximum outbound quota), I will try to merge it shortly and release as v2.0.4

File: clients.go [hunk 1]
```diff
@@ -171,8 +171,63 @@ func NewClient(c net.Conn, o *ops) *Client {
...
+	}
+
+	cl.refreshDeadline(cl.State.keepalive)
+
+	return cl
+}
+
 // NewInlineClient returns a client used when publishing from the embedding system.
-func NewInlineClient(id, remote string) *Client {
+func (s *Server) NewInlineClient(id, remote string) *Client {
```
Comment (mochi-co) [inline clients.go:205]: I like the idea of add new client as a server method, however we shouldn't mix methods and files (eg. server method in client struct file) 👍🏻

Comment (chenji1990) [reply clients.go:205]: I like your code and erudition. I will learn from you more in the future. If there is anything I can do for you, I will always be here.

**Comment by mochi-co** [comment]
Closing as duplicated, by many thanks and fixed in v2.0.4 :)

**Pull request closed without merging**